src/lib/features/feature-toggle/tests/feature-toggle-service.e2e.test.ts

```diff
@@ -814,3 +814,51 @@ test('Should not allow to revive flags to archived projects', async () => {
         ),
     );
 });
+
+test('Should enable disabled strategies on feature environment enabled', async () => {
```

Comment:
Minor suggestion: can we be clearer about what we're checking here? Something like

```suggestion
test('Should not remove constraints or strategy variants on feature environment enabled', async () => {
```


```suggestion
test('Strategy configurations should not change on feature environment enabled', async () => {
```

Reply:
I guess I was late 🤷🏼